docker.go

```diff
@@ -116,6 +120,7 @@ func (docker *dockerConfig) call() int {
 			os.Mkdir(temp, 0755)
 		}
 		dockerArgs = append(dockerArgs, "-v", fmt.Sprintf("%s%s:/var/tgf", convertDrive(tempDrive), tempFolder))
+		config.Environment["TGF_TEMP_FOLDER"] = path.Join(tempDrive, tempFolder)
```

Comment:
This export is used to publish the name of the temporary folder on the host used to execute the actual command.

This is used within the bootstrap to display the complete path relative to the host when entering in a shell `tgf bash`. Then, it is easy to open an editor (VSCode or else) directly in the temporary folder.